docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.5'
+
+services:
+  mifosx-web-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - '4200:80'
+    environment:
+      - FINERACT_API_URLS=https://dev.mifos.io,https://demo.mifos.io,https://qa.mifos.io,https://staging.mifos.io,https://mobile.mifos.io,https://demo.fineract.dev,https://localhost:8443
+      - FINERACT_API_URL=https://localhost:8443
+      - FINERACT_API_PROVIDER=/fineract-provider/api
+      - FINERACT_API_VERSION=/v1
```

Comment:
I don't think we need to separate the URL into 3 pieces cause they'll simply not change independently.
Let's have a single FINERACT_API_URL which is by default https://localhost:8443/fineract-provider/api/v1

Reply:
We had a client that change the Fineract context path for internal policies, then the v1 maybe later could be v2 or something else...

At the end those are base on how they are used in the `environment.ts` with a proper default value each one and with that those are optionals. Even you can run the `docker` image or using `docker-compose` without any of these optional values and you will see the same default settings.